lib/new_google_recaptcha/view_ext.rb

```diff
@@ -3,23 +3,39 @@ module ViewExt
     include ActionView::Helpers::TagHelper
 
     def include_recaptcha_js
-      raw %Q{
-        <script src="https://www.google.com/recaptcha/api.js?render=#{NewGoogleRecaptcha.site_key}"></script>
-      }
+      generate_recaptcha_callback + javascript_include_tag("https://www.google.com/recaptcha/api.js?render=#{NewGoogleRecaptcha.site_key}&onload=newGoogleRecaptchaCallback", defer: true)

```

Comment:
suggest not to use `+`, use "#{expression}" instead.

Reply:
it's ok as for me, we already have interpolation in second statement